### PR TITLE
Fix serialization and refactor virtual switch init logic

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: vslib/src/sai_vs_generic_create.cpp [hunk 1]
```diff
@@ -86,6 +105,8 @@ sai_status_t vs_generic_create(
 {
     SWSS_LOG_ENTER();
 
+    std::string otstr;
```
Comment (qiluo-msft) [inline vslib/src/sai_vs_generic_create.cpp]: Not used.

File: vslib/src/sai_vs_vlan.cpp [hunk 2]
```diff
@@ -78,10 +161,23 @@ sai_status_t vs_remove_vlan_member(
...
+    sai_status_t status = meta_sai_remove_oid(
             SAI_OBJECT_TYPE_VLAN_MEMBER,
             vlan_member_id,
             &vs_generic_remove);
+
+    if (status == SAI_STATUS_SUCCESS)
+    {
+        auto it = vlan_members_map[vlan_id].find(vlan_member_id);
+
+        vlan_members_map[vlan_id].erase(it);
```
Comment (qiluo-msft) [inline vslib/src/sai_vs_vlan.cpp]: Will it be set::end? If never, add assertion?

Comment (kcudnik) [inline vslib/src/sai_vs_vlan.cpp]: no, as you can se its only executed after success, so vlan member id will exists

File: vslib/src/sai_vs_generic_create.cpp (hunk 1 above)
Comment (qiluo-msft) [inline vslib/src/sai_vs_generic_create.cpp]: Closed.

File: syncd/syncd.cpp [hunk 3]
```diff
@@ -1180,21 +1223,21 @@ void handlePortMap(const std::string& portMapFile)
...
-        std::string fp_value = line.substr(0, pos);		
+
+        std::string fp_value = line.substr(0, pos);
         std::string lanes    = line.substr(pos + 1);
         lanes.erase(lanes.begin(), std::find_if(lanes.begin(), lanes.end(), std::not1(std::ptr_fun<int, int>(std::isspace))));
         std::istringstream iss(lanes);
         std::string lane_str;
         std::set<int> lane_set;
-		
+
```
Comment (stcheng) [inline syncd/syncd.cpp]: you could use tokenize() function in swss-common

File: vslib/src/sai_vs_init.cpp [hunk 4]
```diff
@@ -130,135 +127,349 @@ sai_status_t initialize_default_objects()
...
+{
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_INFO("create vlan members for all ports");
+
+    vlan_member_list.clear();
+
+    vlan_members_map[DEFAULT_VLAN_NUMBER] = {};
+
+    for (auto &port_id : port_list)
```
Comment (stcheng) [inline vslib/src/sai_vs_init.cpp]: i doubt if we will add this to vs behavior. default vlan 1 needs to be there, but we could have 0 members in the default vlan. the behavior of adding all ports into default vlan by default is optional.

**Pull request merged**